posthog/helpers/tests/test_session_recording_helpers.py

```diff
@@ -478,9 +478,25 @@ def test_get_events_summary_from_snapshot_data():
                 },
             },
         },
+        # payload has iso string timestamp instead of number and is out of order by timestamp sort
+        # in https://posthog.sentry.io/issues/4089255349/?project=1899813&referrer=slack we saw a client
+        # send this event, which caused the backend sorting to fail because we treat the rrweb timestamp
+        # as if it is always a number
+        {
+            "type": 1,
+            "timestamp": "1987-04-28T17:17:17.590Z",
+            "data": {"source": 3},
+        },
+        # safely ignore string timestamps that aren't timestamps
+        {
+            "type": 1,
+            "timestamp": "it was about a hundred years ago, that I remember this happening",
```

Comment:
Weird stress seeing the order of keys versus the other cases 🥲

Reply:
I've got Friday brain and don't know what you mean 🙉